quantum/errors.js

```diff
@@ -0,0 +1,3 @@
+module.exports.errors = {
```

Comment:
```suggestion
module.exports = {
```
If this file will only hold error message strings, we can remove `errors` here as it's redundant.